docs/history.md

```diff
@@ -1528,7 +1528,7 @@ N/A
 
 * `meteor create --vue` is now available thanks to [@chris-visser](https://github.com/chris-visser). PR [#11086](https://github.com/meteor/meteor/pull/11086)
 
-* `--cache-build` option is now available on `meteor deploy` command and you can use it safely all the time if you are using a Git repository to run your deploy. This is helpful if your upload is failing then you can retry just the upload and also if you deploy the same bundle to multiple environments. [Read more](https://cloud-guide.meteor.com/deploy-guide.html#cache-build).
```

Comment:
The new address for this is: https://galaxy-guide.meteor.com/deploy-command-line.html#cache-build

Reply:
Thanks for your feedback @StorytellerCZ I updated the links accordingly. Have a nice day!